### ISSUE TITLE
Need new openbmc-test-automation IPL test-suites developed for extensive OP HW CI tests

### ISSUE THREAD

**Issue by sameerveer** [openbmc/openbmc-test-automation#1436]
First, some background and then our request:
In our per-commit OP CI flow, we were hit by this latest VPD bug which went undetected for some time  - looks like, you folks can help us here by writing additional test-suites to catch these situations before providing final score back to Jenkins.

As you know, presently,  we do only #1, #2 and #3 (of the below list.)

1) Flash BMC
2) Re-mage PNOR
3) boot with the existing vpd cache
4) clear out the cache (cd /var/lib/phosphor-software-manager/pnor; rm -rf CVPD MVPD DJVPD)
5) boot with empty cache
6) boot using the newly made cache

We require additional test-suites shown above: # 4-6

In summary, we think our OP HW CI should include a boot with the existing vpd, then something wipes it out and we boot fresh.  Otherwise any fresh boot bugs will be hidden for a long time potentially.

Here are the benefit we think, these additional IPLs will realize for us:
3 - catches transition bugs
4/5 - catches cache update bugs
6 - catches bugs related to the cache update we did 

Please advise, thanks and regards,
-Sameer Veer

**Comment by gkeishin** [comment]
Sure @sameerveer  we will take a look into it.

**Comment by ssombar** [comment]
https://gerrit.openbmc-project.xyz/#/c/openbmc/openbmc-test-automation/+/12092
